### DISCUSSION TITLE
Option to disable using Image Optimization , but keep using next/image

### DISCUSSION THREAD

**Discussion by hsnfirdaus** [vercel/next.js#31835] [Ideas]
### Describe the feature you'd like to request

I think it's great idea to have option to enable or disable image optimization for image at export. So we can swtich to use Image Optimization or not, but no need to replace all Image component markup...

### Describe the solution you'd like

Create option in next.config.js to disable or enable image optimization

### Describe alternatives you've considered

Adding parameter on export build to disable image optimization

**Comment by sanjaiyan-dev** [comment]
Use [`Unoptimized` prop on next/image](https://nextjs.org/docs/api-reference/next/image#unoptimized).

**Comment by hsnfirdaus** [comment]
> Use [`Unoptimized` prop on next/image](https://nextjs.org/docs/api-reference/next/image#unoptimized).

Can i use it in export?

**Comment by sanjaiyan-dev** [comment]
> > Use [`Unoptimized` prop on next/image](https://nextjs.org/docs/api-reference/next/image#unoptimized).
> 
> Can i use it in export?

Hi , I am not sure about it but I think *no* because I have seen the same questions raised in other discussions :(

But,after Next js 11.1 they introduced custom loader prop for better `next export` support.So I use this weird technique to use it :)

- In next config js add `images.loader = custom`
- And next js image component I add loader prop -:     `<Image loader={({src,width, quality}) => '${src}?w=${width}'}/>` for [more info](https://nextjs.org/docs/api-reference/next/image#built-in-loaders)
- But I prefer to use image libraries such as `next-optimized-images` for better performance and optimization (if you're working with`next export`)
